assets/js/components/Beneficiaries.vue

```diff
@@ -78,14 +81,18 @@ import ChartMixin from './mixins/Chart';
 import CSVReadingMixin from './mixins/CSVReading';
 import WithFMsMixin from './mixins/WithFMs';
 import WithCountriesMixin, {get_flag_name} from './mixins/WithCountries';
-
+import Dropdown from './includes/DropdownFilter.vue';
 
 export default Vue.extend({
   mixins: [
     BaseMixin, CSVReadingMixin,
-    ChartMixin, WithFMsMixin, WithCountriesMixin,
+    ChartMixin, WithFMsMixin, WithCountriesMixin
   ],
 
+  components: {
```

Comment:
i think it's safe to say all charts will have a dropdown-filter. add it to the `ChartMixin` components instead of repeating it everywhere.